minos/common/model/abc/dto_model.py

```diff
@@ -0,0 +1,43 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from __future__ import (
+    annotations,
+)
+
+import typing as t
+
+from ...protocol import (
+    MinosAvroProtocol,
+)
+from .fields import (
+    ModelField,
+)
+from .model import (
+    MinosModel,
+)
+
+
+class DtoModel(MinosModel):
```

Comment:
I was thinking about a "fancy" name for this class and I think that one alternative could be to name it as `DataTransferModel`. What do you think about that? 🤔

Reply:
Yes, I think it's better to call it that.